src/conn/conn_api.c

```diff
@@ -117,7 +117,7 @@ __conn_get_extension_api(WT_CONNECTION *wt_conn)
 	conn->extension_api.conn = wt_conn;
 	conn->extension_api.err_printf = __wt_ext_err_printf;
 	conn->extension_api.msg_printf = __wt_ext_msg_printf;
-	conn->extension_api.strerror = wiredtiger_strerror;
+	conn->extension_api.strerror = __wt_ext_strerror;
```

Comment:
@michaelcahill, the current Helium/Levyx driver makes a ton of calls into WiredTiger before allocating a WT_SESSION, take a look at [wiredtiger_extension_init()](https://github.com/wiredtiger/wiredtiger/blob/master/ext/datasources/helium/helium.c#L3313).